openmetadata-service/src/main/java/org/openmetadata/service/util/EntityUtil.java

```diff
@@ -347,9 +347,9 @@ public static String getFieldName(String... strings) {
   /** Return column field name of format "columns".columnName.columnFieldName */
   public static <T extends EntityInterface> String getColumnField(
       T entityWithColumns, Column column, String columnField) {
+
     // Remove table FQN from column FQN to get the local name
-    String localColumnName =
-        EntityUtil.getLocalColumnName(entityWithColumns.getFullyQualifiedName(), column.getFullyQualifiedName());
+    String localColumnName = column.getName();
```

Comment:
Hey @sureshms ,
I have a small doubt here why are we using the getLocalColumnName function to get the column name instead we can directly do by column.getName().
as when we are using that function for column children it return the name as parentName."childerName" which breaks further for children.